app/state_machines/workflows/predicates.rb

```diff
@@ -122,6 +122,10 @@ def has_pit_extension?
     @kase.extended_for_pit?
   end
 
+  def case_respondable?   # member of assigned team and outstide escalation period
```

Comment:
Could we name this something like assigned_team_member_outside_escalation_period? I know it's long but it makes the requirement much more clear